bdk-ffi/src/wallet.rs

```diff
@@ -130,6 +130,14 @@ impl Wallet {
             .map_err(CannotConnectError::from)
     }
 
+    pub fn apply_unconfirmed_txs(&self, unconfirmed_txs: Vec<UnconfirmedTx>) {
+        self.get_wallet().apply_unconfirmed_txs(
+            unconfirmed_txs
+                .into_iter()
+                .map(|utx| (Arc::new((&*utx.tx).into()), utx.last_seen)),
```

Comment:
I find this line hard to parse, but I understand it's required. You need to dereference the transaction in the Arc before taking a reference to it. I think in this case using 

```rust
.map(|utx| (Arc::new(utx.tx.as_ref().into()), utx.last_seen)),
```

would work as well and might be easier to read? Just a nit, pick whatever line you like best.

Reply:
Thank you sir, much much easier to read, so thanks for calling this out. Updated